acceptance/repository-mysql/src/__tests__/mysql.transaction.acceptance.ts

```diff
@@ -0,0 +1,24 @@
+// Copyright IBM Corp. 2019. All Rights Reserved.
+// Node module: @loopback/repository-tests
+// This file is licensed under the MIT License.
+// License text available at https://opensource.org/licenses/MIT
+
+import {DefaultCrudRepository} from '@loopback/repository';
+import {
+  CrudRepositoryCtor,
+  transactionTestSuite,
+} from '@loopback/repository-tests';
+import {MYSQL_CONFIG, MYSQL_FEATURES} from './mysql.datasource';
+
+describe('DefaultCrudRepository + mysql connector', () => {
+  transactionTestSuite(
```

Comment:
This is not right. `transactionTestSuite` should be executed as part of running `crudRepositoryTestSuite`, which is already called from [`acceptance/repository-mysql/src/__tests__/mysql-default-repository.acceptance.ts`](https://github.com/strongloop/loopback-next/tree/master/acceptance/repository-mysql/src/__tests__/mysql-default-repository.acceptance.ts) See also [`packages/repository-tests/src/crud-test-suite.ts`](https://github.com/strongloop/loopback-next/blob/master/packages/repository-tests/src/crud-test-suite.ts) which is iterating over all files and automatically running the test suite(s) exported by them.

How to address this issue:
- Remove this file (`mysql.transaction.acceptance.ts`)
- Verify that the transaction-related tests are still executed for MySQL